ModList/Services/ModProviderService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using IPA.Loader;
+using IPA.ModList.BeatSaber.Extensions;
+using IPA.ModList.BeatSaber.Models;
+using Zenject;
+
+namespace IPA.ModList.BeatSaber.Services
+{
+    public class ModProviderService : IInitializable
+    {
+        /// <remark>
+        /// I really didn't want to make it static...
+        /// but due to BSIPA' behaviour regarding how it handles enabling/disabling singleStartInit mods... this really is necessary...
```

Comment:
I'm not sure I understand why this needs to be static. Could you explain?

Reply:
Explained in DMs. Will need more investigation though.